int.ts

```diff
@@ -1706,19 +1706,10 @@ module J2ME {
             var callee = null;
 
             if (isStatic) {
-              object = null;
+              address = Constants.NULL;
```

Comment:
After these changes, the only use of the *object* local in the *interpret()* function is in the *Bytecodes.SWAP* case, which could use the *address* local instead, so we should remove *object* completely.